docs/protobuf.rst

```diff
@@ -2,22 +2,28 @@
-phenopackets-schema.
+Phenopackets schema uses protobuf, an exchange format developed in 2008 by Google. We refer readers to the
+excellent `Wikipedia page on Protobuf <https://en.wikipedia.org/wiki/Protocol_Buffers>`_ and
+to `Google's documentation <https://developers.google.com/protocol-buffers/>`_ for details. This page
+intends to get curious readers who are unfamiliar with protobuf up to speed with the main aspects of this
+technology, but it is not necessary to understand protobuf to use the phenopacket schema.
+
+Google initially developed Protocol Buffers (protobuf) for internal use, but now has provided a code generator for multiple languages under an open source license. In this documentation, we will demonstrate use of phenopackets-schema with Java, but all of the features are available in any of the languages that protobuf works with including C++ and Python. phenopackets-schema uses protobuf version 3.7.0.
 
-Google initially develop Protocol Buffers (protobuf) for internal use, but now has provided a code generator for multiple languages under an open source license. In this documentation, we will demonstrate use of phenopackets-schema with Java, but all of the features are available in any of the languages that protobuf works with including C++ and Python. phenopackets-schema currently uses protobuf version 3.7.0 and we restrict our explanations to protobuf version 3 in this documentation.
 
 The major advantages of protobuf are that it is language-neutral, faster than many other schema languages such as XML and JSON, and can be simpler to use because of features such as automatic validation of data objects.
 
 
-Protobuf forsees that data structures (called ** messages**) are defined in a definition file (with the suffix .proto) and compiled to generate code that is invoked by the sender or recipient of the data to encode/decode the data. 
+Protobuf forsees that data structures (so-called **messages**) are defined in a definition file (with the suffix .proto) and compiled to generate code that is invoked by the sender or recipient of the data to encode/decode the data.
 
 
 ~~~~~~~~~~~~~~~~~~~
 Installing protobuf
 ~~~~~~~~~~~~~~~~~~~
 
-The following exercise is not necessary to use phenopackets-schema, but is intended to build intuition for how protobuf works. We first need to install protobuf. We show one simple way of installing protobuf on a linux system in the following.
+The following exercise is not necessary to use phenopackets-schema,
+but is intended to build intuition for how protobuf works.
+We first need to install protobuf. We show one simple way of installing protobuf on a linux system in the following.
```

Comment:
This isn't true - the `os-maven-plugin` and `protobuf-maven-plugin` will download the correct binary for the system at hand and install it for use in the `target/protoc-plugins` directory.